frontend/src/views/RoleView.vue

```diff
@@ -19,7 +19,11 @@
         <br />
         <ElButton type="primary" @click="showDialogAddNewKnowledge">Add new Knowledge</ElButton>
         <ElDialog v-model="createKnowledgeDialogFormVisible" title="New Knowledge" width="500">
-          <ElInput v-model="newKnowledgeName" placeholder="Knowledge name" :clearable="false" />
+          <ElInput v-model="knowledge.name" placeholder="Knowledge name" :clearable="false" />
+          <div class="SliderBlock">
```

Comment:
You don't have to describe what your class is doing (Block). Your class should just contain the name of your component

Read this to have more info: https://getbem.com/